index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1 class="header">Hello Microverse</h1>
+    <h1 class="header">is one</h1>
+    <h1 class="header">of clear and understandable project</h1>
```

Comment:
You Almost there +1: , But there is some issues you have to check on
I Think h1 must be used only once in page so, maybe you may try to change like using h2, h3, hx